src/components/join/How.jsx

```diff
@@ -1,7 +1,75 @@
+"use client";
+import Image from "next/image";
+import image1 from "@/public/join/how.webp";
+import Link from "next/link";
+import useView from "../useView";
+
 const How = () => {
   return (
-    <div className="">
-      <p>How</p>
+    <div className="ml-7">
+      <p
+        className={`${
+          useView && "animate-fade-right animate-delay-[100ms]"
+        }text-2xl md:text-4xl text-swim-blue-400 font-bold mb-10 `}
```

Comment:
```suggestion
        } text-4xl text-swim-blue-400 font-bold mb-10 `}
```

You can probably use the same font size here.